Containers/talk/Dockerfile

```diff
@@ -1,18 +1,19 @@
 FROM nats:2.9.18-scratch as nats
 FROM strukturag/nextcloud-spreed-signaling:1.1.2 as signaling
-FROM coturn/coturn:4.6.2-r3-alpine
-USER root
-# Pin alpine version manually as long as https://github.com/coturn/coturn/issues/1226 is not done
-ENV ALPINE_VERSION=3.18
+FROM ghcr.io/sando38/eturnal:1.10.1-otp-25 AS eturnal
```

Comment:
I wonder will this be moved back to ghcr.io/processone/eturnal at some point? If yes I would like to wait for that.

Reply:
yes, I just want to see if sando thinks that this could work

Reply:
yes, that will be moved back! Cannot tell you however if it will be in July or August.

Reply:
and maybe the tag changes to `x.x.x-otpxx`

Reply:
https://github.com/nextcloud/all-in-one/pull/2855#discussion_r1240908595